src/main/java/com/github/technus/tectech/thing/metaTileEntity/multi/GT_MetaTileEntity_EM_transformer.java

```diff
@@ -197,6 +195,11 @@ public boolean onRunningTick(ItemStack aStack) {
         return true;
     }
 
+    @Override
```

Comment:
This is just wrong. The explosion is controlled via the `eDismantleBoom` flag. However I'd say the explosion can stay as a whole, so we may wait for other opions on that matter too.

Reply:
better using already existing methods than just messing with overrides. If such  flag exist, then it's better to set it inside the constructor

Reply:
The flag is disabled by default. The constructor of the transformer just sets it to true.